From 7077983d0098d1dfc719dc9f996c88ad6a627180 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 20 Jul 2017 15:43:05 +0800
Subject: [PATCH 275/278] axxia: Added i2c device node for axm5616-victoria
 board

Add i2c device node for axm5616-victoria board, or else the user
app "sensors" can't find sensor via i2c bus.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm64/boot/dts/intel/axm5616-victoria.dts | 52 ++++++++++++++++++++++++++
 1 file changed, 52 insertions(+)

diff --git a/arch/arm64/boot/dts/intel/axm5616-victoria.dts b/arch/arm64/boot/dts/intel/axm5616-victoria.dts
index d7b8ea01..a9a99e0 100644
--- a/arch/arm64/boot/dts/intel/axm5616-victoria.dts
+++ b/arch/arm64/boot/dts/intel/axm5616-victoria.dts
@@ -194,10 +194,51 @@
 
 &i2c0 {
 	status = "okay";
+
+	sysfpga@40 {
+		reg = <0x40>;
+	};
 };
 
 &i2c1 {
 	status = "okay";
+
+	pxa9555@20 {
+		compatible = "pca9555";
+		reg = <0x20>;
+	};
+
+	adt7467@2e {
+		compatible = "adt7473";
+		reg = <0x2e>;
+	};
+
+	temp@18 {
+		compatible = "jc42";
+		reg = <0x18>;
+	};
+
+	temp@1a {
+		compatible = "jc42";
+		reg = <0x1a>;
+	};
+
+	spd@50 {
+		compatible = "spd";
+		reg = <0x50>;
+	};
+
+	spd@52 {
+		compatible = "spd";
+		reg = <0x52>;
+	};
+
+	eeprom@54 {
+		compatible = "24c1024";
+		reg = <0x54>;
+		pagesize = <128>;
+	};
+
 };
 
 &i2c2 {
@@ -206,6 +247,17 @@
 
 &i2c3 {
 	status = "okay";
+
+	ltc2974@5c {
+		compatible = "ltc2974";
+		reg = <0x5c>;
+	};
+
+	ltc2974@5d {
+		compatible = "ltc2974";
+		reg = <0x5d>;
+	};
+
 };
 
 &edac_cpu {
-- 
1.8.5.2.233.g932f7e4

